cmd/cloudinitsave/cloudinitsave.go

```diff
@@ -66,8 +66,10 @@ func Main() {
 		log.Errorf("Failed to save cloud-config: %v", err)
 	}
 
+	// exit wpa_supplicant
+	netconf.StopWpaSupplicant()
```

Comment:
Not sure but if we don't set any WiFi networks, is there a problem with forcibly stopping WPA?

Reply:
Only print log on network container: `Failed to run cmd [wpa_cli terminate]: exist status 255`

Reply:
It's ok.